extensions/podman/src/extension.ts

```diff
@@ -426,8 +426,13 @@ async function registerProviderFor(provider: extensionApi.Provider, machineInfo:
       await startMachine(provider, machineInfo, logger);
     },
     stop: async (context, logger): Promise<void> => {
-      await execPromise(getPodmanCli(), ['machine', 'stop', machineInfo.name], { logger });
-      provider.updateStatus('ready');
+      try {
+        await execPromise(getPodmanCli(), ['machine', 'stop', machineInfo.name], { logger });
+      } catch (err: unknown) {
+        console.error(`Error while stopping Podman machine ${err}`);
```

Comment:
Error outputs to console, but doesn't go anywhere else? Usually we'd return an error / should we add `throw err` here?

Reply:
See https://github.com/containers/podman-desktop/pull/2943#discussion_r1243901076 for the rationale